Descent3/gamesave.cpp

```diff
@@ -622,7 +622,7 @@ bool LoadGameDialog() {
 
     fp = fopen(pathname, "rb");
     if (fp) {
-      int bm_handle;
+      int bm_handle = -1;
       int *pbm_handle;
```

Comment:
it's less important, but pointers should be initialized to `nullptr`

Reply:
It didn't complain about `pbm_handle` being uninitialized. All the code paths that can initialize `pbm_handle` are handled by lines 629-633. If you're referencing the initializer for `pbm_handle` on line 632, I was just handling uninitialized values.